annotation-solr/src/test/resources/solr7-dev/anno-dev/anno-up/conf/schema.xml

```diff
@@ -145,7 +150,7 @@
 			
 		<!-- @Field("body_value.*") NEW -->
 		<dynamicField name="body_value.*" type="text" required="false"
-			multiValued="false" indexed="true" stored="true" />	
+			multiValued="true" indexed="true" stored="true" />	
```

Comment:
we don't have multiple values for individual languages.  This should stay with multivalues=false

Reply:
Changed.